app/police-custody-form/police-custody-form.njk

```diff
@@ -0,0 +1,430 @@
+            idPrefix: "events",
+            name: "events",
+            items: [
+              {
+                value: "PerPropertyChange",
+                text: "Property change",
+                conditional: {
+                  html: propertyChange
+                }
+              }
+            ]
+          }) }}
+          {% endcall %}
+
+          {% set otherDeath %}
+            {{ govukTextarea({
+              id: "PersonMoveDeathInCustody",
+              name: "PersonMoveDeathInCustody",
+              type: "text",
+              classes: "govuk-!-width-two-thirds",
+              label: {
+                html:"<b>Give details</b>"
+              }
+            }) }}
+          {% endset -%}
+
+          {% set otherAnyOtherEvents %}
+            {{ govukTextarea({
+              id: "PerGeneric",
+              name: "PerGeneric",
+              type: "text",
+              classes: "govuk-!-width-two-thirds",
+              label: {
+                html:"<b>Give details</b>"
+              }
+            }) }}
+          {% endset -%}
+
+          {% call govukFieldset({
+            legend: {
+              text: "Other",
+              classes: "govuk-fieldset__legend--m"
+            }
+          }) %}
+
+          {{ govukCheckboxes({
+            idPrefix: "events",
+            name: "events",
+            items: [
+              {
+                value: "PersonMoveDeathInCustody",
+                text: "Death in custody",
+                conditional: {
+                  html: otherDeath
+                }
+              },
+              {
+                value: "PerGeneric",
+                text: "Any other events",
+                conditional: {
+                  html: otherAnyOtherEvents
+                }
+              }
+            ]
+          }) }}
+          {% endcall %}
+
+          <div class="govuk-button-group">
+            {{ govukButton({
+              preventDoubleClick: true,
+              type: submit,
+              text: "Save events"
+            }) }}
+             <input type="hidden" id="moveId" name="moveId" value="{{moveId}}">
+            <a href="/move/{{moveId}}/warnings" class="govuk-link">Cancel</a>
+        </div>
+        </form>
+      </div>
+
+      <aside class="govuk-grid-column-one-third">
+        {% block contentSidebar %}
+
+          {% call _sidebarSection({ first: true }) %}
+            {% include "includes/person-summary.njk" %}
+          {% endcall %}
+
+          {% call _sidebarSection(borderBottomClasses = false) %}
+          <h3 class="govuk-heading-s govuk-!-margin-bottom-2">
+            {{ t("moves::move_details") }}
+          </h3>
+
+          {{ appMetaList(moveDetails) }}
+
+          <p class="govuk-!-margin-top-3 govuk-!-margin-bottom-0 govuk-!-font-size-16">
+            <a href="{{ urls.move.details }}#tabs">
+              {{ t("actions::view_or_edit_move_details") }}
+            </a>
+          </p>
+        {% endcall %}
```

Comment:
I'm wondering if this form should actually be within the `move` app. It has a few benefits, mostly that we can use the same layout template so this wouldn't be required, and it means we can use the same `setMove` middleware.

Reply:
I did think about moving it to the `move` app, and I agree the benefit of the `setMove` middleware. 

I found that the layout for the police custody form page is slightly different, as it has a few less components on the side panel, it doesn't include the links to edit or view move details or the cancel move link. 

I wanted to align it in a similar way to how the person escort record form is laid out in order to have some consistency with these forms.

Reply:
I think in this case then it's worth moving the `setMove` middleware in to the common middlewares since it's useful in multiple apps, and means we can avoid unnecessary duplication.